libraries/rdiff_backup.rb

```diff
@@ -0,0 +1,151 @@
+class Chef
+  class Resource
+    class RdiffBackup < LWRPBase
+      self.resource_name = 'rdiff_backup'
+      actions :create
+      default_action :create
+
+      attribute :owner, kind_of: String, default: 'rdiff-backup-server'
+      attribute :group, kind_of: String, default: 'rdiff-backup-server'
+      attribute :source, kind_of: String, default: '/'
+      attribute :destination, kind_of: String, default: '/data/rdiff-backup'
+      attribute :fqdn, kind_of: String
+      attribute :cookbook, kind_of: String, default: 'rdiff-backup'
+      attribute :remote_user, kind_of: String, default: 'rdiff-backup-client'
+      attribute :ssh_port, kind_of: [String, Integer], default: 22
+      attribute :retention_period, kind_of: String, default: '1W'
+      attribute :args, kind_of: String, default: ''
+      attribute :lock_dir, kind_of: String, default: '/var/rdiff-backup/locks'
```

Comment:
If we use this as a default, then we cannot run concurrent jobs. Was that the intention? If not, we may want to use the resource name as a unique way to lock per job for the files in that directory.